vmm.mk

```diff
@@ -37,6 +37,7 @@ ARCH_INDEP_FILES := src/util/printf.c \
 		    src/virtio/block.c \
 		    src/virtio/console.c \
 		    src/virtio/mmio.c \
+			src/virtio/net.c \
```

Comment:
nitpick sorry, this indentation shows up weird in the diff.

Reply:
Done https://github.com/au-ts/libvmm/pull/95/commits/72d6fb67257d1487a9f61e9bac6d62adaa47315e